Added new term 'glutaminase deficiency' MONDO:0600001

kallia-p opened the pull request:
@nicolevasilevsky Please see below a description of information sources and details of term editing. I have a couple of suggestions for further edits as well and would appreciate your input.

Sources of information for glutaminase deficiency
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6439720/
PMID:30575854
https://pubmed.ncbi.nlm.nih.gov/30970188/
PMID:30970188
Wikipedia wording appears to have been used for the proposed definition with references to the 2 papers above so I annotated the 2 PMIDs above for the definition in MONDO. Pubmed and google search have not come up with any more papers on glutaminase deficiency 
https://en.wikipedia.org/wiki/Glutaminase_deficiency#:~:text=Glutaminase%20deficiency%20is%20a%20rare,results%20in%20an%20early%20demise
 
New term created: 'glutaminase deficiency' MONDO:0600001
I added it as a child of ‘disorder of glutamine metabolism as proposed and also added Subclass of ‘neurometabolic disease’ (it is also referred to in the paper as a neurodevelopmental disease). We may also want to indicate that it is rare (only the 2 publications above report this disorder, total of 7 cases) is autosomal recessive and that it is caused by mutations in the glutaminase (GLS) gene. PMID:30575854 describes a loss-of function mutation whereas PMID:30970188 a mutation in the 5’ region of the gene with less severe phenotype. Please let me know if you think we should add rare, autosomal recessive and that it is caused by mutations in GLS in the Subclass Of field.

To make the suggested changes with regards to the siblings:
MONDO:0034147 neonatal epileptic encephalopathy due to glutaminase deficiency
I added Subclass of ‘glutaminase deficiency’
I suppose we could delete Subclass of ‘disorder of glutamine metabolism’ which is now the parent term of glutaminase deficiency, right? 
 
MONDO:0034146 spastic ataxia-dysarthria due to glutaminase deficiency
I added Subclass of ‘glutaminase deficiency’
As above I suppose we could delete Subclass of ‘disorder of glutamine metabolism’. 
 
MONDO:0032678 developmental and epileptic encephalopathy, 71
I added Subclass of glutaminase deficiency.

nicolevasilevsky commented on src/ontology/mondo-edit.obo:
```diff
@@ -435305,6 +435308,14 @@ xref: GARD:0013029 {source="MONDO:equivalentTo"}
+name: glutaminase deficiency
+def: "Glutaminase deficiency is characterized by refractory seizures, respiratory failure, brain abnormalities and death in the neonatal period, though milder cases with spastic ataxia-dysarthria have also been reported. This condition is caused by mutations in the glutaminase (GLS) gene." [PMID:30575854, PMID:30970188]
+is_a: MONDO:0017352 ! disorder of glutamine metabolism
+is_a: MONDO:0019058 ! neurometabolic disease
```
add the source to the subclassOf axiom, you can add the PMIDs here

nicolevasilevsky commented:
>Please let me know if you think we should add rare, autosomal recessive and that it is caused by mutations in GLS in the Subclass Of field.

Yes, this sounds good, thanks!
Please add sources for these subclassOf axioms too - that you mentioned above

nicolevasilevsky commented:
> I suppose we could delete Subclass of ‘disorder of glutamine metabolism’ which is now the parent term of glutaminase deficiency, right? 

we don't usually delete superclasses in Mondo, because we like to retain the provenance, so we add an 'excluded subclasssOf axiom"

kallia-p commented:
@nicolevasilevsky I committed again with the annotation changes as agreed - all edits or changes are documented below:

**'glutaminase deficiency'**
Added SubClassOf
'disease has basis in dysfunction of' some GLS | 'disorder of glutamine metabolism' |  'has modifier' some rare | 'neurometabolic disease' | 'inborn errors of metabolism' (I annotated PMID:30575854 and PMID:30970188 as sources)

 **'spastic ataxia-dysarthria due to glutaminase deficiency'** - 
Changed SubclassOf assertion: has material basis in gain of function germline mutation in' some GLS was changed to Subclass Of: 'disease has basis in dysfunction of' some GLS  and I annotated sources (Orphanet:557056 and PMID:30970188)
I excluded subClassOf "http://purl.obolibrary.org/obo/MONDO_0017352" i.e. 'disorder of glutamine metabolism' and I annotated the Orphanet source.
I added SubclassOf 'has modifier' some rare | 'inborn errors of metabolism' | 'neurometabolic disease' and I annotated both sources as above (after checking that this info is in Orphanet and in the paper).   

**'neonatal epileptic encephalopathy due to glutaminase deficiency'** 
Changed SubclassOf assertion: has material basis in gain of function germline mutation in' some GLS was changed to SubclassOf: 'disease has basis in dysfunction of' some GLS  and I annotated sources (Orphanet:557064 and PMID:30575854)
I excluded subClassOf "http://purl.obolibrary.org/obo/MONDO_0017352" i.e. 'disorder of glutamine metabolism' and I annotated the Orphanet source.
I added other subclassesOf: 'has modifier' some rare |'inborn errors of metabolism' |'neurometabolic disease' and I annotated sources as above (after checking that this info is in Orphanet and in the paper) so all subclasses have both annotations.


**Important To Check** I noticed that the other child of glutaminase deficiency 'developmental and epileptic encephalopathy, 71' OMIM:618328 has an xref to the same Orphanet term as above. When I checked the OMIM source the PMID is the same PMID:30575854. So OMIM and Orphanet must refer to the same thing and I wonder if these terms need to be merged?

nicolevasilevsky commented:
> Important To Check I noticed that the other child of glutaminase deficiency 'developmental and epileptic encephalopathy, 71' OMIM:618328 has an xref to the same Orphanet term as above. When I checked the OMIM source the PMID is the same PMID:30575854. So OMIM and Orphanet must refer to the same thing and I wonder if these terms need to be merged?

Very good catch @kallia-p!
I agree these should be merged.

One thing that can be kind of confusing with Orphanet is sometimes they do BTNT (broad to narrow term) mappings to OMIM, but in this case, it is an E (exact) mapping - i can see that here:
https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_557064

Could you please create a ticket using this [template](https://github.com/monarch-initiative/mondo/issues/new?assignees=nicolevasilevsky&labels=merge&template=mergeTerm.md&title=%5BMerge%5D)

I collect a [list](https://docs.google.com/spreadsheets/d/1tt1Wk70j9XiHLV1vKQyNiHhaazh286pobpJk1ecSCCg/edit#gid=319577054) of terms to be merged and email the user list and give them at least 2 weeks to respond before doing any mergers (or obsoletions)
